controller/src/beerocks/master/db/db.cpp

```diff
@@ -232,6 +232,35 @@ int db::get_node_channel(std::string mac)
     return n->channel;
 }
 
+int db::get_hostap_operating_class(const sMacAddr &mac)
+{
+    auto mac_str = network_utils::mac_to_string(mac);
+    auto n       = get_node(mac_str);
+    if (!n) {
+        LOG(WARNING) << __FUNCTION__ << " - node " << mac_str << " does not exist!";
+        return 0;
+    } else if (n->get_type() != beerocks::TYPE_SLAVE || n->hostap == nullptr) {
+        LOG(WARNING) << __FUNCTION__ << "node " << mac_str << " is not a valid hostap!";
```

Comment:
```suggestion
        LOG(WARNING) << "node " << mac_str << " is not a valid hostap!";
```

`LOG()` already prints the function if so configured. The default config doesn't print the function name, but since it prints file and line number it's not needed either.

Same comment below as well.

Reply:
Set function was removed as not needed.